code/datums/materials/_material.dm

```diff
@@ -41,7 +41,7 @@ Simple datum which is instanced once per type and is used for every object of sa
 		source.name = "[name] [source.name]"
 
 	if(beauty_modifier)
-		addtimer(CALLBACK(source, /datum.proc/AddComponent, /datum/component/beauty, beauty_modifier * amount), 0)
+		addtimer(CALLBACK(source, /datum.proc/_AddComponent, list(/datum/component/beauty, beauty_modifier * amount)), 0)
```

Comment:
Why use a time and not this?
`AddComponent(/datum/component/beauty, (beauty_modifier * amount))`

Reply:
You're asking the wrong person, changing this would be out of scope for this pr especially considering how many possible ways just this pr could break things.